pkg/endpoint/api.go

```diff
@@ -95,7 +95,6 @@ func NewEndpointFromChangeModel(ctx context.Context, owner regeneration.Owner, p
 	ep.aliveCancel = cancel
```

Comment:
Nit: I believe the commit message is incorrect, in that each endpoint has its own controller manager; even if different endpoints have the same controller name, they will not override each other. It does make reading output of `cilium status --all-controllers` confusing, though.

Reply:
Ah honestly I thought it would override. I'll amend the commit message.